plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/edit/MySQLDatabaseManager.java

```diff
@@ -64,8 +64,14 @@ protected MySQLCatalog createDatabaseObject(DBRProgressMonitor monitor, DBEComma
     protected void addObjectCreateActions(DBRProgressMonitor monitor, DBCExecutionContext executionContext, List<DBEPersistAction> actions, ObjectCreateCommand command, Map<String, Object> options)
     {
         final MySQLCatalog catalog = command.getObject();
-        final StringBuilder script = new StringBuilder("CREATE SCHEMA `" + catalog.getName() + "`");
-        appendDatabaseModifiers(catalog, script);
+        final StringBuilder script;
```

Comment:
Please reformat this code please.
Because we are not using quotes anymore, we have a special function.
So, it will be 

```java
String catalogName = DBUtils.getQuotedIdentifier(catalog);
...
new StringBuilder("CREATE DATABASE " + catalogName);
...
new StringBuilder("CREATE SCHEMA " + catalogName);
```